bundles/org.openhab.binding.xmltv/src/main/feature/feature.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<features name="org.openhab.binding.xmltv-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
-    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+<features name="org.openhab.binding.xmltv-${project.version}"
+	xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
```

Comment:
Tags are preferred to be on a single line in feature files. Unfortunately spotless put the on a separate line. There is a fix for that, but not yet applied here. So could you put this features rag on a single line.

Reply:
Yes, sure and this indeed was caused by the spotless formatter.